tests/foreman/cli/test_repository.py

```diff
@@ -1790,6 +1793,158 @@ def test_positive_upload_content_srpm(self):
+        """Check module-stream get with list on hammer.
+
+         :id: 9842a0c3-8532-4b16-a00a-534fc3b0a776ff89f23e-cd00-4d20-84d3-add0ea24abf8
+
+         :Setup:
+             1. valid yum repo with Module Streams.
+         :Steps:
+             1. Create Yum Repositories with url contain module-streams and Products
+             2. Initialize synchronization
+             3. Verify the module-stream list with various inputs options
+
+         :expectedresults: Verify the module-stream list response.
+
+         :CaseAutomation: automated
+         """
+        repo1 = self._make_repository({
+            u'content-type': u'yum',
+            u'url': CUSTOM_MODULE_STREAM_REPO_1,
+        })
+        Repository.synchronize({'id': repo1['id']})
+        product2 = make_product_wait(
+            {u'organization-id': self.org['id']},
+        )
+        repo2 = self._make_repository({
+            u'content-type': u'yum',
+            u'url': CUSTOM_MODULE_STREAM_REPO_2,
+            u'product-id': product2['id']
+        })
+        Repository.synchronize({'id': repo2['id']})
+        module_streams = ModuleStream.list()
+        self.assertEqual(len(module_streams), 12,
+                         'Module Streams get worked Correctly')
+        module_streams = ModuleStream.list({'product-id': product2['id']})
+        self.assertEqual(len(module_streams), 7,
+                         'Module Streams get worked Correctly')
+
+    @tier1
+    def test_module_stream_info_validation(self):
+        """Check module-stream get with info on hammer.
+
+         :id: ddbeb49e-d292-4dc4-8fb9-e9b768acc441a2c2e797-02b7-4b12-9f95-cffc93254198
+
+         :Setup:
+             1. valid yum repo with Module Streams.
+         :Steps:
+             1. Create Yum Repositories with url contain module-streams
+             2. Initialize synchronization
+             3. Verify the module-stream info with various inputs options
+
+         :expectedresults: Verify the module-stream info response.
+
+         :CaseAutomation: automated
+         """
+        repo1 = self._make_repository({
+            u'content-type': u'yum',
+            u'url': CUSTOM_MODULE_STREAM_REPO_1,
+        })
+        Repository.synchronize({'id': repo1['id']})
+        product2 = make_product_wait(
+            {u'organization-id': self.org['id']},
+        )
+        repo2 = self._make_repository({
+            u'content-type': u'yum',
+            u'url': CUSTOM_MODULE_STREAM_REPO_1,
+            u'product-id': product2['id']
+        })
+        Repository.synchronize({'id': repo2['id']})
+        actual_result = ModuleStream.info({u'id': "12"})
+        expected_result = {
+            'module-stream-name': 'nodejs',
+            'stream': '8',
+            'architecture': 'noarch',
+            'artifacts': [
+                {
+                    'id': '27',
+                    'name': 'nodejs-1:8.10.0-3.module_1572+d7ec111e.x86_64'
```

Comment:
If we use such values, we might need to update testcase everytime. Can we have some better way?

Reply:
@ntkathole not everytime unless until there change in modulemd file or change in the repo, which supposed to be infrastructure change can be controlled. let me know your thoughts?

Reply:
@ntkathole @pondrejk I understood that i need to make sure that repos should be hosted somewhere in QA infra.